.github/actions/release/run.sh

```diff
@@ -82,9 +83,9 @@ do
         # Additionally grab the step number of the 'sha256sum' step
         step=$(curl --silent "https://api.github.com/repos/dfinity/internet-identity/actions/runs/$GITHUB_RUN_ID/jobs" \
             | jq -cMr \
-            --arg job_id "$job_id" \
+            --argjson job_id "$job_id" \
```

Comment:
Turns out this change actually broke the `jq` command due to types. `argjson` parses the value as `number` whereas `arg` always parses to `string`